mod/discussions/start.php

```diff
@@ -31,7 +31,8 @@ function discussion_init() {
 	elgg_register_plugin_hook_handler('register', 'menu:owner_block', 'discussion_owner_block_menu');
 
 	// Register for search.
-	elgg_register_plugin_hook_handler('search', 'object:discussion', 'discussion_search_discussion');
+	elgg_register_plugin_hook_handler('search:options', 'object:discussion', 'discussion_search_discussion');
+	elgg_register_plugin_hook_handler('search_types', 'get_queries', 'discussion_filter_search_types');
```

Comment:
can't we change this while we are breaking? or trigger the old hook in a new hook?